openwisp_users/api/serializers.py

```diff
@@ -0,0 +1,402 @@
+import logging
+
+from allauth.account.models import EmailAddress
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Permission
+from django.db import transaction
+from django.db.models import Q
+from django.utils.translation import ugettext_lazy as _
+from openwisp_utils.api.serializers import ValidatedModelSerializer
+from rest_framework import serializers
+from swapper import load_model
+
+Group = load_model('openwisp_users', 'Group')
+Organization = load_model('openwisp_users', 'Organization')
+User = get_user_model()
+OrganizationUser = load_model('openwisp_users', 'OrganizationUser')
+logger = logging.getLogger(__name__)
+OrganizationOwner = load_model('openwisp_users', 'OrganizationOwner')
+
+
+class OrganizationSerializer(ValidatedModelSerializer):
+    class Meta:
+        model = Organization
+        fields = (
+            'id',
+            'name',
+            'is_active',
+            'slug',
+            'description',
+            'email',
+            'url',
+            'created',
+            'modified',
+        )
+
+
+class CustomPrimaryKeyRelatedField(serializers.PrimaryKeyRelatedField):
+    def display_value(self, instance):
+        return instance.user.username
+
+    def get_queryset(self):
+        user = self.context['request'].user
+        if user.is_superuser:
+            queryset = OrganizationUser.objects.all()
+        else:
+            queryset = OrganizationUser.objects.filter(
+                Q(organization__in=user.organizations_managed)
+            )
+        return queryset
+
+
+class OrganizationOwnerSerializer(serializers.ModelSerializer):
+    organization_user = CustomPrimaryKeyRelatedField(allow_null=True)
+
+    class Meta:
+        model = OrganizationOwner
+        fields = ('organization_user',)
+        extra_kwargs = {'organization_user': {'allow_null': True}}
+
+
+class OrganizationDetailSerializer(serializers.ModelSerializer):
+    owner = OrganizationOwnerSerializer(required=False)
+
+    class Meta:
+        model = Organization
+        fields = (
+            'id',
+            'name',
+            'is_active',
+            'slug',
+            'description',
+            'email',
+            'url',
+            'owner',
+            'created',
+            'modified',
+        )
+
+    def update(self, instance, validated_data):
```

Comment:
Things look orthodox, except this function would need some testing.
@ManishShah120 I hope you have already done functional testing, as done before in other PRs! 😄